### PR TITLE
Add CORS headers to responses

### PR DESCRIPTION
**Pull request by the-sageman**
Nothing too interesting, just that people now can't use browser JavaScript on other websites to post on fchannel

### CODE REVIEW

**Comment by FChannel0** [comment]
What are the benefits for enabling CORS?
From my understanding it allows you to use scripts with a different host than the origin. What benefit for FChannel does this have? Shouldnt the origin be the only one able to execute scripts to itself? 

What is having CORS disabled atm doing differently than if CORS was enabled? 

This PR could change potential security issues, so it should be discussed before merging.

**Comment by the-sageman** [comment]
> 
> 
> What are the benefits for enabling CORS?
> From my understanding it allows you to use scripts with a different host than the origin. What benefit for FChannel does this have? Shouldnt the origin be the only one able to execute scripts to itself?
> 
> What is having CORS disabled atm doing differently than if CORS was enabled?

The original intent of this PR was to block requests from foreign scripts on other sites, but now that I read about it a little bit more, same origin policy is on by default so actually and CORS only acts as a whitelist for that, so it actually makes FChannel worse security-wise. The only use it _might_ have is in remote administration from another website, but that's not really the main intent of this PR.
In short: This is kind of an embarassing PR for me😬 I apologize for wasting your time.
